### PR TITLE
Add CompartiBike

### PR DESCRIPTION
**Pull request by edumucelli**
Adding CompartiBike, 3 cities in Brazil.

### CODE REVIEW

File: pybikes/data/compartibike.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "instances":[
+       {
+            "tag":"integrabike",
+            "meta":{
+                "latitude":-23.501719,
+                "city":"Sorocaba",
+                "name":"IntegraBike Sorocaba",
```
Comment (eskerda) [inline pybikes/data/compartibike.json]: Sorocaba is an implicit part of the name (city). Also, it appears the name is `Integrabike`.

File: pybikes/compartibike.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            if latitude == 1 and longitude == 1:
+                continue
+
+            name = station['name']
+            bikes = int(station['available_slots_size'])
+            free = int(station['unavailable_slots_size'])
+
+            extra = {
+                'uid': int(station['id']),
+                'open': station['status'] == 'Ativa'
```
Comment (eskerda) [inline pybikes/compartibike.py]: More fields that could be added to extra:
- `number` (`station_number`): the human id of the station, sometimes it appears on names).
- `bike_uids` (`bikes.bike_hard_id` or `bikes.id`): there's even more info available, but at least for now we can track bike uids.

File: pybikes/compartibike.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            longitude = float(station['googleMapY'])
+            latitude = float(station['googleMapX'])
+
+            # Skip "test" stations
+            if latitude == 1 and longitude == 1:
+                continue
+
+            name = station['name']
+            bikes = int(station['available_slots_size'])
+            free = int(station['unavailable_slots_size'])
```
Comment (eskerda) [inline pybikes/compartibike.py]: Is this right? I think `unavailable_slots_size` are bikes, not slots. Likewise, `available_slots_size` are slots.

File: pybikes/compartibike.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+            # Skip "Loldesign" stations
+            if station['googleMapY'] == "" or station['googleMapX'] == "":
+                continue
+
+            longitude = float(station['googleMapY'])
+            latitude = float(station['googleMapX'])
+
+            # Skip "test" stations
+            if latitude == 1 and longitude == 1:
+                continue
```
Comment (eskerda) [inline pybikes/compartibike.py]: We can do more clever filters using a bounding box. See [pybikes/nextbike.py](https://github.com/eskerda/pybikes/blob/master/pybikes/nextbike.py).

**Comment by edumucelli** [comment]
@eskerda, I have applied all required modifications.

File: pybikes/data/compartibike.json [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+            },
+            "feed_url":"https://integrabike.compartibike.com.br/stations.json"
+        },
+        {
+            "tag":"rivibike",
+            "meta":{
+                "latitude":-23.501719,
+                "city":"São Lourenço",
+                "name":"RiviBike",
+                "longitude":-47.45079,
```
Comment (eskerda) [inline pybikes/data/compartibike.json]: Sure these lat/lngs are right? This looks the same as Sorocaba.

**Comment by eskerda** [comment]
Merged in https://github.com/eskerda/pybikes/pull/217

Thanks!

**Pull request closed without merging**